contracts/solidity/account/NonExisting.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+contract NonExisting {
+    function callOnNonExistingAccount(address nonExistingAddr) external {
+        nonExistingAddr.call{value: 0}(
+            abi.encodeWithSignature("doesNotExist()")
+        );
+    }
+
+    function delegatecallOnNoneExistingAccount(address nonExistingAddr) external {
+        (bool success, bytes memory data) = nonExistingAddr.delegatecall(
```

Comment:
we can remove `(bool success, bytes memory data)` if it's not used in the function